src/Autofac.Extras.AttributeMetadata/Autofac.Extras.AttributeMetadata.csproj

```diff
@@ -43,8 +43,8 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Autofac, Version=3.4.0.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Autofac.3.4.0\lib\net40\Autofac.dll</HintPath>
+    <Reference Include="Autofac, Version=4.9.4.0, Culture=neutral, PublicKeyToken=17863af14b0044da, processorArchitecture=MSIL">
```

Comment:
Is there a reason this has to change _other_ than this is the version of the core library you're using? Reason I ask is that we generally try to keep the _lowest_ version possible as a reference for compatibility reasons, at least in the distributed library. It's OK if the unit tests get upgraded separately.

Reply:
I thought if there is no bug in the 3.4.0 version there is no reason to fix it. Maybe we should upgrade the version of library and add restriction to use Autofac.Extras.AtributeMetadata only with Autofac lower than 4.9.0? 
For now I reverted all references. All tests were passed

Reply:
Right - if the unit test assembly references the new Autofac we can verify that the bug is fixed; but the product assembly can still reference the old Autofac for backwards compatibility.